### DISCUSSION TITLE
Stop Widgets from being changed into a different layout

### DISCUSSION THREAD

**Discussion by maheshrajrp** [eclipse-theia/theia#10446] [Q&A] [answered]
Hi,
Assume I'm having a widget placed in bottom shell. How do I stop from being moved to a different shell say `main` ?

Thanks

**Comment by vince-fugnitto** [comment, marked as answer]
@maheshrajrp thank you for the discussion, I believe that the question was asked previously, please take a look at https://community.theia-ide.org/t/turn-off-docking-abilities/1426/2.

**Reply by maheshrajrp** [reply]
Thanks for your answer. I'll work on that.
